app/assets/javascripts/src/Metamaps.Map.js

```diff
@@ -49,12 +49,18 @@ Metamaps.Map = {
       return false
     })
 
+    $('.starMap').click(function () {
+      if ($(this).text() === 'star') self.star()
+      else self.unstar()
```

Comment:
Lol, yeah this is super hacky. I coded this whole feature in two hours on Saturday night, and rushed a few little things, knowing that there'd be a few changes to make when I  get an image asset for the starring/unstarring. I will do something to improve it at that point :)